### ISSUE TITLE
Koodikatselmointi 2

### ISSUE THREAD

**Issue by heltsu** [CarnivoreBarnacle/Forum#2]
Hei

Tässä hiukan ajatuksia joita heräsi kun katselin koodiasi ja yritin tutustua tekemääsi sovellukseen.
Yritin kovasti loggautua sisälle, mutta en onnistunut. Saattaa johtua myös omasta kömpelyydestäni. Olisi ollut ihan mielenkiintoista nähdä minkälaiset sivut olet rakentanut, mutta kenties näen niitä sitten deomssa.
Koodia yritin katsoa ja se näytti minusta varsin selkeältä ja hyvin rakennetulta. Metodien nimet olivat selkeitä. Virheilmoitukset olivat hyvin kuvaavia. Päivämäärät olit saanut hienosti käännettyä "meikäläiseen" muotoon. 
Käyttöohjetta yritin löytää, mutta se on varmaankin vielä valmistumassa ja odottamassa julkaisua parempana ajankohtana. 
Kaiken kaikkiaan siisti ja selkeä kokonaisuus.

**Comment by CarnivoreBarnacle** [comment]
Joo, näköjään unohtui lähettää päivitetty tietokanta users-palvelimelle, minkä takia sisäänkrijautuminen ei onnistunut. Nyt pitäisi toimia. Kiitos palautteesta.
